lib/zendesk_apps_support/validations/secure_settings.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ZendeskAppsSupport
+  module Validations
+    module SecureSettings
+      SECURABLE_KEYWORDS = %w[token key pwd password].freeze
+      SECURABLE_KEYWORDS_REGEXP = /#{ Regexp.union(SECURABLE_KEYWORDS).source }/i
```

Comment:
I'm sad that there's no nicer way to do this than to compile a regex, unwrap it into a string, then compile that into another regex. I guess you could do something like `Regexp.new(SECURABLE_KEYWORDS.join('|'), Regexp::IGNORECASE)` - not sure if that's nicer.

Reply:
Sounds good to me!